### PR TITLE
Hk add proxy protocol usage

### PR DESCRIPTION
**Pull request by KebirHakim**

### CODE REVIEW

File: pages/public_cloud/containers_orchestration/managed_kubernetes/Octavia_Tutorial_Add_proxy_protocol/guide.en-gb.md
Comment (Jessica41) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/Octavia_Tutorial_Add_proxy_protocol/guide.en-gb.md:1]: Hello @KebirHakim before we begin the proofread can you please adjust the extra spacing that is present for each code blocks? If I start eliminating the extra spacing, I will have trouble adjusting the `nginx` code blocks. The ```bash ones can be done or left untouched, they are easier to adjust.

**Comment by Y0Coss** [comment]
@KebirHakim , the title of the PR seems unrelated to its actual content?
Why is the Rancher guide also included in this PR ?

You also need to create meta.yaml files + update the index

**Comment by scraly** [comment]
About the Rancher guide, @KebirHakim I think you should create another PR specially for that guide and don't forget to add it in the index file in "Creating or importing a Kubernetes cluster" category. Thx :)

**Comment by KebirHakim** [comment]
Sorry for the mistake, I fixed all the points. I created a new PR better structured and with the meta+yaml file and the index

**Comment by Y0Coss** [comment]
> Sorry for the mistake, I fixed all the points. I created a new PR better structured and with the meta+yaml file and the index

I don't see any changes on this Pull Request

**Comment by Y0Coss** [comment]
@KebirHakim, I'm closing this PR as the Load Balancer content is now handled on https://github.com/ovh/docs/pull/6704 
The Rancher guide should be handled with a different branch and PR.

**Pull request closed without merging**